fabric/build.gradle

```diff
@@ -18,7 +18,7 @@ buildscript {
          */
 
         dependencies {
-            classpath "com.github.kristofbolyai:EventBusTransformer:3fedef0169"
```

Comment:
It's just weird that a central dependency -- that we develop ourselves -- should reside in personal name space.
I forked your repo into https://github.com/Wynntils/EventBusTransformer, so you can switch to that instead.